modules/acid/src/main/ts/ephox/acid/api/colour/HexColour.ts

```diff
@@ -8,27 +8,21 @@ const hexColour = (value: string): Hex => ({
 const shorthandRegex = /^#?([a-f\d])([a-f\d])([a-f\d])$/i;
 const longformRegex = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i;
 
-const isHexString = (hex: string): boolean => {
-  return shorthandRegex.test(hex) || longformRegex.test(hex);
-};
+const isHexString = (hex: string): boolean => shorthandRegex.test(hex) || longformRegex.test(hex);
```

Comment:
This I like

Reply:
I think that change came from the arrow-body-style lint in the previous PR, I will check.

Reply:
Confirmed.